### ISSUE TITLE
Photo page link text

### ISSUE THREAD

**Issue by mvl22** [cyclestreets/mobileweb2011#21]
"_Full link to photo page_ (for tweeting etc.)"
is a bit wordy. I suggest

"_Tweet this_ | _Share on Facebook_ | _Permalink_"

with the URLs being

http://twitter.com/?status=Great+photo+on+%40CycleStreets%21+http%3A%2F%2Fwww.cyclestreets.net%2Flocation%2F<id>%2F
(i.e. Great photo on @CycleStreets! http://www.cyclestreets.net/location/<id>/ )

http://www.facebook.com/sharer/sharer.php?u=http%3A%2F%2Fwww.cyclestreets.net%2Flocation%2F<id>%2F

http://www.cyclestreets.net/location/<id>/

Where <id> is the photo ID obviously.

We could even have nice 16x16 icons by making a copy from:
http://www.cyclestreets.net/images/icons/extras/facebook.png
http://www.cyclestreets.net/images/icons/extras/twitter.png

Note that http://www.cyclestreets.net/ rather than m.cyclestreets.net should be used, as we always want to make the www the primary link target, as www will then decide whether the user is mobile or not.

**Comment by annapowellsmith** [comment]
All nice and clear - fixed and pushed, looks good I think.

**Comment by mvl22** [comment]
Stupidly I failed to give the correct URL for the Twitter one.

Tweet this needs to use:
 http://cycle.st/p<id>

This will redirect to www.cyclestreets.net/location/<id>/ which will then decide whether to redirect to mobile or not. But the point is that the cycle.st short name has a nicer appearance in twitter, and the presence of '@cyclestreets' means that our 'brandname' appears in the tweet anyway.

**Comment by si-the-pie** [comment]
Fixd in...

https://github.com/cyclestreets/mobileweb/commit/3f16768900276141f846f953d04892fb5f0e8c85
